storage/bootstrap/bootstrapper/fs/source.go

```diff
@@ -38,13 +39,15 @@ type newFileSetReaderFn func(
 	filePathPrefix string,
 	readerBufferSize int,
 	bytesPool pool.CheckedBytesPool,
+	decodingOpts msgpack.DecodingOptions,
 ) fs.FileSetReader
 
 type fileSystemSource struct {
 	opts             Options
 	log              xlog.Logger
 	filePathPrefix   string
 	readerBufferSize int
+	decodingOpts     msgpack.DecodingOptions
```

Comment:
Seems like this never gets set from the constructor?  Should `bootstrapper/fs/Options` perhaps get a `DecodingOptions()` and `SetDecodingOptions(value) Options` methods?